README.md

```diff
@@ -117,11 +119,6 @@ GenRMQ.Publisher.publish(Publisher, Jason.encode!(%{msg: "msg"}))
 - [Consumer without deadletter configuration][without_deadletter_configuration]
 - [Consumer with quorum queues][with_quorum_queue_type]
 
-### Metrics
```

Comment:
I am not convinced with removing this documentation. We should still treat these modules as internal ones. Users should not interact with them directly meaning that they might not come across their documentation.

Reply:
Good point! I added it back in the README and have the links going to the modules. That work?

Reply:
@akoutmos yes, sounds good 👍